package/src/specs/NativeSkiaModule.ts

```diff
@@ -0,0 +1,8 @@
+import type { TurboModule } from "react-native/Libraries/TurboModule/RCTExport";
+import { TurboModuleRegistry } from "react-native";
```

Comment:
here we probably want to use  to use platform (`Platform` is what we use to "polyfill" react native API on web without the need to use react-native-web)